lib/web_ui/test/engine/platform_dispatcher/platform_dispatcher_test.dart

```diff
@@ -210,10 +210,11 @@ void testMain() {
       final EngineFlutterView view22 =
           EngineFlutterView(22, dispatcher, createDomHTMLDivElement());
 
-      dispatcher
-        ..registerView(view20)
-        ..registerView(view21)
-        ..registerView(view22);
+      // Add this again when views don't register themselves upon instantiation.
+      // dispatcher
+      //   ..registerView(view20)
+      //   ..registerView(view21)
+      //   ..registerView(view22);
```

Comment:
/cc @mdebbar I had to remove this temporarily, because of...

Reply:
Ah yes, thank you! Please don't delete them though. I'll be uncommenting them when I move registration out of the `EngineFlutterView` constructor.